go/src/reader/reader.go

```diff
@@ -1,51 +1,69 @@
 package reader
 
-import "regexp"
-import "fmt"
+import (
+	"fmt"
+	"regexp"
+
+	"types"
+)
 
 func Read_str(input string) string {
-  fmt.Println(tokenizer(input));
-  return "ASdf";
+	fmt.Println(tokenizer(input))
+	return "ASdf"
```

Comment:
All of these extra spaces are actually tabs. `gofmt` uses tabs for indentation instead of spaces.